javascript/src/V1_LEGACY/streaming-basic/live-from-microphone.js

```
@@ -1,19 +1,24 @@
-import { exit } from "process";
-import WebSocket from "ws";
-import mic from "mic";
+"use strict";
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)